src/web/components/elements/UnsafeEmbedBlockComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { css } from 'emotion';
+
+type Props = {
+    html: string;
+    alt: string;
+    index: number;
+};
+
+const fullWidthStyles = css`
+    width: 100%;
+`;
+export const UnsafeEmbedBlockComponent = ({ html, alt, index }: Props) => (
+    <iframe
+        className={`${fullWidthStyles} atom__iframe`}
+        title={alt}
+        name={`unsafe-embed-${index}`}
```

Comment:
Can you add a comment for what this is used for? Tripped me up for a sec, but it's the 'contract' required for the iframe script, right? Along with the `atom__iframe` class - if you can add a comment so that that's clear for the future, that'd be great!

Reply:
Have been thinking we should have a docs directory called `contracts` with all the basic info on these types of things.

Reply:
you need a unique identifying name for the iFrame, it is used in the script here: https://github.com/guardian/dotcom-rendering/pull/1946/files#diff-e500d36de993a83661c36277af8605152cc33bd2aa0d03b7f5b1563655c4d805R10

Reply:
I know yeah, I meant a comment in the code for future people :D